File: x2024/c/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);
    for (int i = 0; i < height; i++)
    {
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: x2024/python/mario-less/mario.py
from cs50 import get_int

while True:
    size = get_int("Height: ")
    if size > 0 and size <= 8:
        break

for i in range(size):
    for j in range(size-1, i, -1):
        print(" ", end="")
    for k in range(0, i+1):
        print("#", end="")
    print()
